### PR TITLE
CSS and JS update of the chat plugin

### PR DESCRIPTION
**Pull request by thomasgx**
Now, all work with the class `y_active` set on the main div `yelda_container`.
Display of the iframe, of the bubble and the minimal notifications depends on the  `y_active` presence.

### CODE REVIEW

**Comment by meelie** [comment]
@reviewer : the purpose of this PR is to be 
- reviewed
- tested 
once validated, published on npm with S3 upload (uploadBuildToS3)

and a new PR should be opened on yelda with
- npm update
- new version of the injectors in the frontend/static folder

thanks !

File: src/js/yeldaWebchatInjector.js [hunk 1]
```diff
@@ -277,6 +279,43 @@ class YeldaChat {
...
+      /**
+       * If we are sure that the webchat should be opened from the creation and displayed all the time
+       * we add the y_active class to webChatContainer at the creation
+       *
+       * Otherwise yelda_assistant_img click event management will take care of showing and hiding the webchat
+       * - closeChat() remove "y_active" from the yelda container
+       * - openChat() add y_active" to the yelda container
+       */
+     if (shoulBeOpened && cantBeClosed) {
+       this.webChatContainer.classList.add('y_active')
```
Comment (vishnu-ravi) [inline src/js/yeldaWebchatInjector.js:315]: class names can be moved to config

Comment (meelie) [reply src/js/yeldaWebchatInjector.js:315]: indeed, good suggestion.
multiple classes are manipulated in the injector, so i instead open an issue on which we could work on when will we have more time.
https://github.com/Yeldaai/chat-plugin/issues/112

**Review by meelie — APPROVED** [review]

**Review by vishnu-ravi — APPROVED** [review]

**Comment by meelie** [comment]
new version of the plugin has been deployed

**Pull request merged**